submissions/sprint_b/app.py

```diff
@@ -0,0 +1,29 @@
+from . import app
+from flask import jsonify
+from .helpers import EpithetGenerator, Vocabulary
+
+
+@app.route("/")
+def generate_epithets():
+    epithet_generator = EpithetGenerator()
+    final_string = epithet_generator.generate_epithets()
+
+    return jsonify({"epithets": final_string})
+
+@app.route("/<int:insults>")
+def insults(insults):
```

Comment:
app.py:14:12: W0621: Redefining name 'insults' from outer scope (line 14)

Your function and argument have the same identifer.